documentation/modules/deploying/con-securing-kafka-connect-api.adoc

```diff
@@ -0,0 +1,66 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deploy-kafka-connect-with-plugins.adoc
+
+[id='con-securing-kafka-connect-api-{context}']
+= Limiting access to the Kafka Connect API
+
+[role="_abstract"]
+To grant a user access to the Kafka Connect API, a Kafka cluster admin must provide them with the API URL. 
+This includes the hostname or IP address of the Kafka Connect cluster and the port number on which the API is running (by default, port 8083). 
+Additionally, the user must have authenticated access to the Kubernetes cluster and the Kafka Connect REST API endpoint in order to make a request.
+
+It is crucial to restrict access to the Kafka Connect API only to trusted users to prevent unauthorized actions and potential security issues. 
+The Kafka Connect API provides extensive capabilities for altering connector configurations, which makes it all the more important to take security precautions.
+
+If you are using the `KafkaConnector` custom resource to manage connectors, Kubernetes RBAC rules limit permission to make changes to Kubernetes cluster administrators by default.
```

Comment:
I found this a little difficult to parse. I think the "by default" being at the end doesn't help:

```suggestion
If you are using the `KafkaConnector` custom resources then by default the Kubernetes RBAC rules permit only Kubernetes cluster administrators to make changes to connectors.
```